README.md

```diff
@@ -30,11 +30,16 @@ npm run start
 
 ## Configuration
 
-In order to set up your url you need to create `.env` file in the root:
+In order to set up your application you need to create `.env` file in the root:
 
 ```javascript
 /**
  * .env
  */
 REACT_APP_BASE_URL='http://your-url.com'
+REACT_APP_CLIENT_ID='your-client-id'
+REACT_APP_SCOPE='read:org'
+REACT_APP_EXCHANGE_CODE_ON_TOKEN_URL='https://your-url.com/auth/github/token'
```

Comment:
I would replace this by `REACT_APP_GITHUB_TOKEN_URL`

Reply:
👍